test/integration/options/posixExitCodes.spec.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var helpers = require('../helpers');
+var runMocha = helpers.runMocha;
+
+describe('--posix-exit-codes', function () {
+  // subprocess
+  var mocha;
+
+  function killSubprocess() {
+    mocha.kill('SIGKILL');
+  }
+
+  // these two handlers deal with a ctrl-c on command-line
+  before(function () {
+    process.on('SIGINT', killSubprocess);
+  });
+
+  after(function () {
+    process.removeListener('SIGINT', killSubprocess);
+  });
```

Comment:
[Question] I don't follow, why include these two handlers? If I remove them locally this test still passes.

Reply:
Unfortunately, I've lost the context on why these were originally added; it does seem like they can be removed. 👍

Reply:
Now that I think about it, these were most likely added as a developer convenience to be able to kill the mocha process if there was a problem in the signal handling test suite that caused it to hang (eg. subprocess not responding to the signal as intended). These handlers would allow the developer to ctrl-c out of the mocha process without having to go to another shell prompt, find the PID and manually kill it.  I wanted to make sure that if this test suite didn't work as intended on someone else's operating system, they would still be able to ctrl-c out of the test suite and get back to their command line, but I think that risk is small enough that they're not essential.